src/app/public/modules/wait/wait-adapter.service.ts

```diff
@@ -37,7 +33,7 @@ export class SkyWaitAdapterService implements OnDestroy {
     isFullPage: boolean,
     isWaiting: boolean,
     isNonBlocking = false,
-    waitCmp: SkyWaitComponent = undefined
+    id?: string
```

Comment:
Can we go with `waitComponentId` to be a bit more descriptive?

Reply:
done